incubator/hnc/Makefile

```diff
@@ -30,17 +30,32 @@ test: build
 # Builds all binaries (manager and kubectl) and manifests
 build: generate fmt vet manifests
 	go build -o bin/manager ./cmd/manager/main.go
-	go build -o bin/kubectl-hnc ./cmd/kubectl/main.go
+	go build -o bin/kubectl/kubectl-hnc ./cmd/kubectl/main.go
 
+# Clean all binaries (manager and kubectl)
+clean:
+	rm -r bin/kubectl/kubectl-hnc
+	rm -f bin/manager
+	
 # Install kubectl plugin
 kubectl: build
-	go build -o ${GOPATH}/bin/kubectl-hnc ./cmd/kubectl/main.go
+	cp bin/kubectl/kubectl-hnc ${GOPATH}/bin/kubectl-hnc
 	@echo "Installed kubectl-hnc to GOPATH/bin"
 
 # Run against the configured Kubernetes cluster in ~/.kube/config
 run: build
 	go run ./cmd/manager/main.go --novalidation
 
+# Install kubectl plugin locally using krew.
+# Require to archive bin/kubectl/kubectl-hnc binary with name hnc.tar.gz, then 
+# do sha256sum plugin/hnc.tar.gz. Copy the checksum into plugin/hnc.yaml
```

Comment:
This should all be automated. Can you add a krew-build that does all this?

Reply:
Ok. Will push new one after solve other nits.